### ISSUE TITLE
Number of likes in Feeds are inaccurate

### ISSUE THREAD

**Issue by SpriterGors** [bluesky-social/social-app#6968]
### Steps to Reproduce

1. Browse a user;
2. Visit their "Feeds" tab;
3. Notice that the feed has a number of likes;
4. Click on it and see that the number of likes is not the same as in the Feeds tab.

### Attachments

1: User's (my) Feeds tab displaying "Liked by 135 users"
![image](https://github.com/user-attachments/assets/68ce9bf2-c37f-44d6-9f09-9b7cbb035695)

2: Feed page displaying "Liked by **136** users"
![image](https://github.com/user-attachments/assets/3b2f5da0-bbb8-4f72-81c4-9bb683e748f9)

### What platform(s) does this occur on?

Android, Web (Desktop), Web (Mobile)

### Device Info

Windows 10 Pro

### What version of the app are you using?

Build version: 1.95.0; Bundle info: 1170261 (prod); Bundle date: 24120421; Platform: web; Platform version: undefined

### Additional Information

If the Feed has zero likes, it works as intended (both displays zero likes)

![image](https://github.com/user-attachments/assets/928a4f8d-1e05-48bb-a1a4-0b354ef0295b)

![image](https://github.com/user-attachments/assets/3337524a-b096-46ae-a52a-59017e5decd2)

**Comment by surfdude29** [comment]
Yeah I've noticed this too, the first one doesn't seem to count your own like.
